griffon/commands/entities/osidb.py

```diff
@@ -532,6 +537,218 @@ def delete_flaw_references(ctx, flaw_id, reference_uuid, **params):
     return cprint(data, ctx=ctx)
 
 
+# flaw acknowledgments
+@flaws.group(
+    help=f"{OSIDB_API_URL}/osidb/api/v1/flaws/<id>/acknowledgments", name="acknowledgments"
+)
+@click.pass_context
+def flaw_acknowledgments(ctx):
+    """OSIDB Flaw Acknowledgment."""
+    pass
+
+
+@flaw_acknowledgments.command(name="get")
+@click.option(
+    "--flaw-id",
+    "flaw_id",
+    help="Flaw CVE-ID or UUID.",
+    required=True,
+)
+@click.option("--uuid", "acknowledgment_uuid", help="Acknowledgment UUID.", required=True)
+@query_params_options(
+    entity="Flaw Acknowledgment",
+    endpoint_module=osidb_api_v1_flaws_acknowledgments_retrieve,
+    options_overrides={
+        "include_fields": {"type": click.Choice(OSIDBService.get_fields(FlawAcknowledgment))},
+        "exclude_fields": {"type": click.Choice(OSIDBService.get_fields(FlawAcknowledgment))},
+        "include_meta_attr": {
+            "type": click.Choice(OSIDBService.get_meta_attr_fields(FlawAcknowledgment))
+        },
+    },
+)
+@click.pass_context
+@progress_bar
+def get_flaw_acknowledgment(ctx, flaw_id, acknowledgment_uuid, **params):
+    params = multivalue_params_to_csv(params)
+
+    session = OSIDBService.create_session()
+    data = session.flaws.acknowledgments.retrieve(flaw_id, acknowledgment_uuid, **params)
+    return cprint(data, ctx=ctx)
+
+
+@flaw_acknowledgments.command(name="list")
+@click.option(
+    "--flaw-id",
+    "flaw_id",
+    help="Flaw CVE-ID or UUID.",
+    required=True,
+)
+@query_params_options(
+    entity="Flaw Acknowledgment",
+    endpoint_module=osidb_api_v1_flaws_acknowledgments_list,
+    options_overrides={
+        "include_fields": {"type": click.Choice(OSIDBService.get_fields(FlawAcknowledgment))},
+        "exclude_fields": {"type": click.Choice(OSIDBService.get_fields(FlawAcknowledgment))},
+        "include_meta_attr": {
+            "type": click.Choice(OSIDBService.get_meta_attr_fields(FlawAcknowledgment))
+        },
+    },
+)
+@click.pass_context
+@progress_bar
+def list_flaw_acknowledgments(ctx, flaw_id, **params):
+    # TODO: handle pagination
+    # TODO: handle output
+    session = OSIDBService.create_session()
+
+    params = multivalue_params_to_csv(params)
+    data = session.flaws.acknowledgments.retrieve_list(flaw_id, **params).results
```

Comment:
why not async option ?

Reply:
See the TODO above, basically we have to to deal with 2 things, and this is common for all the list views. How to handle pagination? Does it makes sense so display all at once? And second, how to handle include/exclude fields, we need to account for dynamic fields. I would probably do that together.